.github/workflows/python.yml

```diff
@@ -65,9 +65,9 @@ jobs:
         uses: actions/cache@v3
         with:
           path: .tox/
-          key: ${{ runner.os }}-tox-v3-${{ hashFiles('**/setup.cfg') }}
+          key: ${{ runner.os }}-tox-v4-${{ hashFiles('**/setup.cfg') }}
           restore-keys: |
-            ${{ runner.os }}-tox-v3-
+            ${{ runner.os }}-tox-v4-
```

Comment:
I removed it in https://github.com/sourmash-bio/sourmash/pull/2393/files#diff-092659a9bd0068791326fb1d08f005532e5aeb983b999a7bd32c51deee0a71d5L64-L71
but that looks good too =]

Reply:
it's helpful to have it in there for cases like this...